test/vast_tracker.js

```diff
@@ -285,18 +285,47 @@ describe('VASTTracker', function() {
         });
       });
 
-      describe('#errorWithCode', () => {
+      describe('#error', () => {
```

Comment:
As we are getting rid of the mocha test can you write them in the jest file `vast_trackers.spec.js` please ? 😄